control_toolbox/src/pid.cpp

```diff
@@ -406,9 +401,16 @@ double Pid::compute_command(double error, double error_dot, const double & dt_s)
     cmd_unsat_ = p_term + i_term_ + d_term;
   }
 
-  if (gains.saturation_)
+  if (std::isfinite(gains.u_min_) || std::isfinite(gains.u_max_))
```

Comment:
do we need this check here? shouldn't that be done in the set_gains method or parameter callback?

Reply:
I just added it here and if somehow it passes the traditional set_gains

Reply:
ok